doc/classes/OS.xml

```diff
@@ -667,7 +690,7 @@
 			<argument index="0" name="enable" type="bool">
 			</argument>
 			<description>
-				Set to true to enable the low cpu usage mode. In this mode, the screen only redraws when there are changes, and a considerable sleep time is inserted between frames. This way, editors using the engine UI only use very little cpu.
+				Sets to true to enable the low cpu usage mode. In this mode, the screen only redraws when there are changes, and a considerable sleep time is inserted between frames. This way, editors using the engine UI only use very little cpu.
```

Comment:
This one should be second person, because it sets the mode, but you "Set to [code]true[/code]".

Change the last sentence to:
" Use this in tool mode to reduce CPU usage."